lib/src/judo_numeric_input.dart

```diff
@@ -23,8 +23,8 @@ class JudoNumericInput extends StatefulWidget {
     this.inCard = false,
     this.hidden = false,
     this.order,
-    this.precision,
-    this.scale,
+    this.precision = 10,
+    this.scale = 2,
```

Comment:
Scale should be `0` by default since if we are facing with a custom type which actually has a scale, it'll be set. Otherwise prevent potential validation errors.

Reply:
+1